src/test.sh

```diff
@@ -0,0 +1,4 @@
+export PYTHONPATH="${PYTHONPATH}:$HOME/3rdparty/TheanoNetCore/TheanoNetCoresrc"
```

Comment:
@paroj Thanks, you are right. :+1: 

If I wanted to feed in a single RGB-D patch (cropped and more ore less centered), where in the code would this be most appropriate to do? I would like to use your pipeline to obtain a camera-relative pose , acc. score and class id for images, as depicted below.

How sensitive do you think is the scale/size of the object contained in the patch?

Thanks for your support!
duboisf@student.ethz.ch

![color2](https://cloud.githubusercontent.com/assets/15896008/25371670/20d28452-2992-11e7-9701-ba19f67aac22.png)